bedrock/mozorg/templates/mozorg/home/includes/ctd-promo-de.html

```diff
@@ -0,0 +1,61 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% block content %}
+  <section class="ctd-promo">
+
+    <!-- CTD Promo for non-Firefox users -->
+    <div class="ctd-promo-non-fx">
+      {% call split(
+        block_class='mzp-t-split-nospace mzp-l-split-center-on-sm-md t-pocket-promo',
```

Comment:
Guessing the `t-pocket-promo` is a leftover?

Reply:
oop!